### DISCUSSION TITLE
How to Delete Github Pages Deployment History ?

### DISCUSSION THREAD

**Discussion by casual-coderr** [community/community#30350] [Other Feature Feedback, Questions, & Ideas]
Is there anyway to delete Github pages deployment history ?
![Screenshot 2022-08-22 at 10 36 33 AM](https://user-images.githubusercontent.com/93610399/185843410-da250e4d-2b77-4830-a996-5a2005a698fb.png)

**Comment by radar-blip** [comment]
I think there is a way to hide it. 
Or maybe delete it completely, you would have to read here (if you want) :
https://stackoverflow.com/questions/63236766/how-to-remove-github-pages-from-environments-in-github-repository
